scripts/babel-relay-plugin/src/__fixtures__/fragmentRequiredArgs.fixture

```diff
@@ -0,0 +1,17 @@
+Input:
+var Relay = require('react-relay');
+var x = Relay.QL`
+  fragment on Viewer {
+    actorByName() {
```

Comment:
Ah great catch.  In the fixture I updated the field call and moved the `relay(pattern: true)` directive to the `fragment on Viewer` level.